common/build-style/python3-module.sh

```diff
@@ -24,15 +24,20 @@ do_build() {
 }
 
 do_check() {
-	if [ -z "$make_check_target" ]; then
-		if ! python3 setup.py --help test >/dev/null 2>&1; then
-			msg_warn "No command 'test' defined by setup.py.\n"
-			return 0
+	if python3 -m pytest --help >/dev/null 2>&1; then
+		python3 -m pytest ${make_check_args} ${make_check_target}
+	else
```

Comment:
I'm not against leaving a historical comment like "we don't try to use tox because ...."